### DISCUSSION TITLE
Configuring MinIO Tenants with Dual Storage Classes (NFS and Longhorn) for Bucket-Level Storage Management on Kubernetes

### DISCUSSION THREAD

**Discussion by lohith-vunet** [minio/minio#20700] [Q&A]
I am deploying minio tenant on kubernetes env.I have 2 storage classes (longhorn and nfs).For storing contents of one particular bucket i need to use nfs storage class and for storing all other buckets content i need to use longhorn storage class.

**Comment by klauspost** [comment]
That is not possible, and neither storage backend is recommended anyway.

**Reply by lohith-vunet** [reply]
Hi klauspost ,Can you please suggest any other ways to achieve the above ??

some particular number of buckets to use one type of storage class and other particular number of buckets to use another type of storage class.

**Reply by klauspost** [reply]
@lohith-vunet How does "not possible" imply there are "other ways to achieve the above"?
